packages/diffhtml-components/lib/component.js

```diff
@@ -1,7 +1,7 @@
+import process from './util/process';
```

Comment:
Can't we import `./util/process` as `import { process } from 'diffhtml-shared-internals';`? There are a couple imports throughout that can be rewritten this way, which may help solve some of the issues you're having.